### DISCUSSION TITLE
Getting the error: Failed to send message. HTTP 429

### DISCUSSION THREAD

**Discussion by jpsowin** [danny-avila/LibreChat#1154] [Troubleshooting]
I keep getting this error whenever I try to chat. I'm using Docker and it's not hitting OpenAI, the error seems to be coming from within the docker instance. 

`Something went wrong. Here's the specific error message we encountered: Failed to send message. HTTP 429 - {"error": "slow down"}`

I tried multiple OpenAI models. They key says it hasn't been used on OpenAI dashboard.

**Comment by danny-avila** [comment]
![image](https://github.com/danny-avila/LibreChat/assets/110412045/44b2b34e-5d0e-4229-b35a-f974df008ee0)

I'm having trouble, too. ive been getting error 429 - {"error": "slow down"}, the last hour

They've been having outages all day: https://status.openai.com/

**Reply by jpsowin** [reply]
Ah, I thought that was resolved. Appreciate the response!

**Reply by danny-avila** [reply]
Actually, after sleeping on this, and still getting the issue, it seems they started blocking how specific requests are shaped. I'm working on a fix #1157
